### PR TITLE
[CBRD-23844] Revised sql answers for user schema

### PR DESCRIPTION
**Pull request by zionyun**
http://jira.cubrid.org/browse/CBRD-24195

### CODE REVIEW

**Review by youngjinj — APPROVED** [review]

File: medium/_01_fixed/answers/f_agg1.answer [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ===================================================
-class faculty1    max(ssn)    
+class joe.faculty1    max(ssn)    
 null     55
```
Comment (youngjinj) [inline medium/_01_fixed/answers/f_agg1.answer:3]: It should be the result below.
```
  class joe.faculty1       max(ssn)
===================================
  joe.faculty1                   55
```

Comment (youngjinj) [inline medium/_01_fixed/answers/f_agg1.answer:3]: When I execute the query, the result is not null, so why is the result of TC null?
```
  class joe.faculty1       max(ssn)
===================================
  joe.faculty1                   55
```
Even if I execute it in develop, the result of TC is null.

File: medium/_01_fixed/cases/fview1.sql [hunk 2]
```diff
@@ -1,17 +1,17 @@
 autocommit off;
 call login('joe') on class db_user;
-create class xxx (a int);
-insert into xxx values(6);
-create vclass xxx_v (a1 int) as select a from xxx;
+create class joe.xxx (a int);
+insert into joe.xxx values(6);
+create vclass joe.xxx_v (a1 int) as select a from joe.xxx;
```
Comment (youngjinj) [inline medium/_01_fixed/cases/fview1.sql]: No need to specify owner while creating.
Also, an owner does not need to be specified when using an owned object.
It should be changed as below.
```
create class xxx (a int);
insert into xxx values(6);
create vclass xxx_v (a1 int) as select a from xxx;
```

File: medium/_01_fixed/cases/fview11.sql [hunk 3]
```diff
@@ -1,11 +1,11 @@
 autocommit off;
 select product_code, descr, price
-from all inventory_v i
+from all joe.inventory_v i
 order by descr asc;
   select product.product_code, product.descr, product.price
```
Comment (youngjinj) [inline medium/_01_fixed/cases/fview11.sql]: Please match the indentation as well.

Comment (zionyun) [reply medium/_01_fixed/cases/fview11.sql]: reflected. please check.

File: medium/_01_fixed/cases/fview1.sql (hunk 2 above)
Comment (zionyun) [reply medium/_01_fixed/cases/fview1.sql]: reflected. please check.

File: medium/_01_fixed/answers/f_agg1.answer (hunk 1 above)
Comment (zionyun) [reply medium/_01_fixed/answers/f_agg1.answer:3]: I don't know why null occurs in that TC. When first written, it is expected to have a purpose.

File: medium/_01_fixed/cases/fview1.sql (hunk 2 above)
Comment (youngjinj) [reply medium/_01_fixed/cases/fview1.sql]: Thank you.

File: medium/_01_fixed/cases/fview11.sql (hunk 3 above)
Comment (youngjinj) [reply medium/_01_fixed/cases/fview11.sql]: Thank you.

File: medium/_01_fixed/answers/f_agg1.answer (hunk 1 above)
Comment (youngjinj) [reply medium/_01_fixed/answers/f_agg1.answer:3]: Thank you. I hope it gets fixed later.

**Pull request merged**